src/interface/src/app/plan/create-scenarios/constraints-panel/constraints-panel.component.html

```diff
@@ -0,0 +1,133 @@
+<div class="constraints-panel">
+  <!-- Header -->
+  <div class="flex-row-space-between">
+    <h1>Set Constraints</h1>
+    <mat-chip-list>
+      <mat-chip>
+        <mat-chip-avatar>
+          <mat-icon class="material-symbols-outlined">info</mat-icon>
+        </mat-chip-avatar>
+        Required
+      </mat-chip>
+    </mat-chip-list>
+  </div>
+
+  <!-- Description -->
+  <p class="constraints-description">
+    Constraints will be used to inform project areas and scenario options
+  </p>
+
+  <!-- Form -->
+  <div class="constraints-content">
+    <form class="constraints-form" [formGroup]="constraintsForm">
+      <!-- Budget -->
+      <div formGroupName="budgetForm" required>
+        <label class="form-label required">Budget</label>
+        <mat-radio-group formControlName="optimizeBudget" required>
+          <!-- Custom Budget -->
+            <mat-radio-button [value]="false">
+              <!-- Min -->
```

Comment:
The min and max fields don't appear in the screenshot you took. Were they added after?

Reply:
Oops I took a screenshot with it scrolled down. Updated!